### PR TITLE
feat: clear depend on property list cache when remote list get deleted

### PR DESCRIPTION
**Pull request by chinnu-focaloid**
Related to https://oat-sa.atlassian.net/browse/ADF-487
Related PR : https://github.com/oat-sa/tao-core/pull/3099/files
**Description**
Clear depend on property list cache when remote list get deleted

### CODE REVIEW

File: model/lists/ListService.php [hunk 1]
```diff
@@ -96,7 +97,14 @@ public function removeList(RdfClass $listClass)
             new ValueCollectionDeleteInput($listClass->getUri())
         );
 
-
+        if ($this->isRemote($listClass)) {
+            $this->getDependsOnPropertyRepository()->deleteCache(
+                [
+                            'listUri' => $listClass->getUri()
```
Comment (gabrielfs7) [inline model/lists/ListService.php:103]: fix indentation, please

Comment (gabrielfs7) [reply model/lists/ListService.php:103]: but we will remove this cache here either way...

**Review by boajer — APPROVED** [review]

**Comment by boajer** [comment]
@chinnu-focaloid 
Just a reminder
- [ ] Adjust `tao-core` version composer.json before merging this

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Comment by gabrielfs7** [comment]
- [ ] Update tao-core version after tag is released https://github.com/oat-sa/tao-core/actions/runs/1252710876

**Pull request merged**